tests/routes_tests/test_ballots.py

```diff
@@ -390,6 +402,67 @@ def test_ab_audit_ballot_happy_path(
         assert ballots[0] == {**ballot, **audit_request}
 
 
+def test_ab_audit_ballot_overvote(
+    client: FlaskClient,
+    election_id: str,
+    jurisdiction_ids: List[str],
+    contest_ids: List[str],
+    round_1_id: str,
+    audit_board_round_1_ids: List[str],
+):
+    set_logged_in_user(client, UserType.AUDIT_BOARD, audit_board_round_1_ids[0])
+    rv = client.get(
+        f"/election/{election_id}/jurisdiction/{jurisdiction_ids[0]}/round/{round_1_id}/audit-board/{audit_board_round_1_ids[0]}/ballots"
+    )
+    ballots = json.loads(rv.data)["ballots"]
+    ballot_id = ballots[0]["id"]
+
+    contest_id = contest_ids[1]
+    contest = Contest.query.get(contest_id)
+    choice_ids = [c.id for c in contest.choices]
+    assert len(choice_ids) > contest.votes_allowed
+
+    rv = put_json(
+        client,
+        f"/election/{election_id}/jurisdiction/{jurisdiction_ids[0]}/round/{round_1_id}/audit-board/{audit_board_round_1_ids[0]}/ballots/{ballot_id}",
+        {
+            "status": "AUDITED",
+            "interpretations": [
+                {
+                    "contestId": contest_id,
+                    "interpretation": "VOTE",
+                    "choiceIds": choice_ids,
+                    "comment": None,
+                },
+            ],
+        },
+    )
+    assert_ok(rv)
+
+    interpretation = BallotInterpretation.query.get((ballot_id, contest_id))
+    assert interpretation.is_overvote
+
```

Comment:
With the addition of overvote ballots in the database that now show a status of `AUDITED`, I wonder if there are any code paths that are going to count those in tallies. Is there any test we can add here with a submitted overvote that shows that it is *not* being taken into account in any tallies?